src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -419,7 +417,7 @@
   </ItemGroup>
   <Import Project="shared\System.Private.CoreLib.Shared.projitems" />
   <PropertyGroup>
-    <CheckCDefines Condition="'$(CheckCDefines)'==''">true</CheckCDefines>
+    <CheckCDefines Condition="'$(CheckCDefines)'=='' And '$(PYTHON)' != ''">true</CheckCDefines>
```

Comment:
It seems like silently avoiding checks if python is not installed is incorrect. My opinion would be to allow it to fail, and we create a follow up future work item to convert the check to msbuild.

Reply:
Mmm. I don't need to make this change at all. Its really just a convenience as I prefer for it to be possible to compile major projects in the tree without specifying any special parameters to the msbuild command. I'll remove this adjustment.